docs/source/customization.rst

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Ah, I got `plone.restapi` mixed up with `plone.rest`, which already has an (alpha) release. :+1: for no CL from me :wink: